src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs

```diff
@@ -153,6 +153,22 @@ public class SetAzureNetworkWatcherConfigFlowLogCommand : NetworkWatcherBaseCmdl
         [ValidateRange(1, int.MaxValue)]
         public int RetentionInDays { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Type of flow log format.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter("Json")]
```

Comment:
@zwswim is there a plan to add more types in the future?

Reply:
Yes, suppose to add more type in the future